src/problems/120.triangle.jl

```diff
@@ -0,0 +1,42 @@
+#
+# @lc app=leetcode id=120 lang=julia
+#
+# [120] Triangle
+#
+#=
+Given a triangle, find the minimum path sum from top to bottom. Each step you may move to adjacent numbers on the row below.
+
+For example, given the following triangle
+
+[
+     [2],
+    [3,4],
+   [6,5,7],
+  [4,1,8,3]
+]
+The minimum path sum from top to bottom is 11 (i.e., 2 + 3 + 5 + 1 = 11).
+
+Note:
+
+Bonus point if you are able to do this using only O(n) extra space, where n is the total number of rows in the triangle.
+
+=#
+# @lc code=start
+function minimumTotal(triangle::Vector{Vector{Int}}) 
```

Comment:
Let's stick with the BlueStyle to use lower case with underscors for method names. https://github.com/invenia/BlueStyle#synopsis